ni_measurementlink_service/_internal/parameter/serializer.py

```diff
@@ -158,15 +159,15 @@ def _get_overlapping_parameters(
             )
         type = parameter_metadata_dict[field_index].type
         is_repeated = parameter_metadata_dict[field_index].repeated
-        decoder = serialization_strategy.Context.get_decoder(type, is_repeated)
+        decoder = serialization_strategy.get_decoder(type, is_repeated)
         inner_decoder = decoder(field_index, field_index)
         parameter_bytes_io = BytesIO(parameter_bytes)
         parameter_bytes_memory_view = parameter_bytes_io.getbuffer()
         position = inner_decoder(
             parameter_bytes_memory_view,
             position + encoder._TagSize(field_index),  # type: ignore[attr-defined]
             len(parameter_bytes),
-            type,
+            cast(Message, None),  # unused
```

Comment:
This is another situation where we are assuming that new_default is never used, and if it is used, customers will get a cryptic error about `NoneType`.

Would it be better to pass a dummy message? (The answer may be "no")

Reply:
Clarified the comment as opposed to a dummy message. The parameter can be passed back to us in _new_default, but we're the ones that aren't using the parameter.